tools/build/binary-release/Linux/README.md

```diff
@@ -0,0 +1,99 @@
+# Rakudo Perl 6
+
+This is a pre-built package of Rakudo Perl 6, a Perl 6 compiler.
+
+This package includes the Rakudo Perl 6 compiler and the module installer Zef.
+
+
+## Running Perl 6
+
+To run a Perl 6 program, open a command prompt and type
+
+    /path/to/this/folder/bin/perl6 my_script.p6
+
+or start a REPL by calling `perl6` without an argument
+
+    /path/to/this/folder/bin/perl6
+
+To add the relevant paths to your environment so you don't have to type the full path run the following script:
+
+    source scripts/set-env.sh
+
+
+## Installing modules
+
+To install Perl 6 modules you can use the Zef module installer.
+
+    /path/to/this/folder/bin/perl6 /path/to/this/folder/share/perl6/site/bin/zef install JSON::Fast
+
+Modules will be installed into this Perl 6 package and will thus be available even when moving this package.
+
+
+### Native code modules
+
+To install modules that require a compiler toolchain, you have to have a compiler installed.
+
+- On Debian/Ubuntu based systems do `sudo apt-get install gcc make`
+- On RedHat/Fedora based systems do `sudo dnf install gcc make`
+
+
+## Changes
+
+Recent changes and feature additions are documented in the `docs/ChangeLog`
```

Comment:
FWIW this will likely change to some extent. See https://github.com/perl6/problem-solving/issues/53. @kawaii is working on a proposal. But for this PR it is probably OK as it is.